docs/releases/2.0.txt

```diff
@@ -250,6 +254,14 @@ Database backend API
   ``datetime_extract_sql()`` methods now return only the SQL to perform the
   operation instead of SQL and a list of parameters.
 
+* Increased the number of rows fetched by :meth:`.QuerySet.iterator` at each
+  iteration from 100 to 2000. The old behavior can be restored using the
+  ``chunk_size`` parameter. For instance, considering a ``Book`` model:
```

Comment:
I'm curious to know how the django highlighter is different from python. It might be for the django template language, actually. I've seen html+django as a highlighter.